fix(curriculum): improve wording (#44605)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/step-041.md

Use the `button` element to create a clickable button. For example, `<button>Click Here</button>` creates a button with the text `Click Here`.

> Add a `button` element with the text `Submit` below the `input` element. Note the default behavior of clicking a form button without any attributes submits the form to the location specified in the form's `action` attribute.

Reviewer: ```suggestion
Add a `button` element with the text `Submit` below the `input` element. The default behavior for clicking a form button without any attributes submits the form to the location specified in the form's `action` attribute.
```
Although this was not initially included in this PR, I would personally prefer to phrase the second sentence in this text block this way. The word "Note" at the beginning of the sentence is not needed and makes the sentence harder to read.

Author: I agree, or add a comma after "Note", but removing it works. But I think "for" should still be changed to "of". What do you think? This is my first contribution to any repo, so I'm in a major learning mode.

Author: I removed 'Note', capitalized 'the' but left my original replacement of 'for' with 'of'.

Author: Who needs to resolve this conversation? Me or import-brain?

Author: I missed a change for Step 58. I just changed "last" to "previous".

Reviewer: > Who needs to resolve this conversation? Me or import-brain?

I do not have the ability to resolve conversations on this PR, please resolve this, it looks good :)

Reviewer: > I agree, or add a comma after "Note", but removing it works. But I think "for" should still be changed to "of". What do you think? This is my first contribution to any repo, so I'm in a major learning mode.

"For" just sits better in my head than "of", but this is probably personal preference.

Author: I'm new at this...do you want me to change it back to "for"? And it looks like I need two approvals. Should I wait to change it back to "for" until the approval? Also, was my message too long and it should have been `fix(curriculum): improve wording`? I can use `fix(curriculum)` going forward if that is preferred. What is the next step for me?

Reviewer: > I'm new at this...do you want me to change it back to "for"? And it looks like I need two approvals. Should I wait to change it back to "for" until the approval? Also, was my message too long and it should have been `fix(curriculum): improve wording`? I can use `fix(curriculum)` going forward if that is preferred. What is the next step for me?

No worries mate, we were all new once :)

1. Using "for" or "of" would be personal preference, in the grand scheme of things, it's an extremely minor phrasing change, I'm just giving my two cents :)
2. This repo's PRs are set up to require approval from two reviewers with write access to the repository. I am not a contributor with write access, so although I sent an approving review, my name is not listed in the "At least 2 approving reviews are required by reviewers with write access." section of the Checks.
3. FreeCodeCamp recommends the Conventional Commits commit specification for titling PR titles and commit messages, which is explained [here](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title). Generally, commit messages and PR titles should capture the gist of what you're submitting and be less than thirty characters. Further elaboration should be included in the description.

Please ask me if you have any more questions, the wording might've been a bit confusing. Hope this helps!

Author: I'll just wait then. 

So the commit message should be less than 30 characters? That's not a lot. By description, do you mean in the text field of the Pull Request form? Before the checkboxes? No need to reply if that is correct.

I made some notes on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) but I'm still not exactly sure what to choose. Since I'm just starting out, I'm working on typos for the beta version of the Responsive Web Design course. Thanks!

Reviewer 2: The title of your PR should be nice and short. Then in the actual body of the PR, you can provide a longer description. Hope that clears it up.
